CoursesManager.UI/Database/Procedures/Students/spStudents_Edit.sql

```diff
@@ -0,0 +1,30 @@
+DELIMITER $$
+CREATE DEFINER=`courses_manager`@`%` PROCEDURE `spStudents_Edit`(
+    IN p_id INT,
+    IN p_first_name VARCHAR(255),
+    IN p_last_name VARCHAR(255),
+    IN p_email VARCHAR(255),
+    IN p_phone VARCHAR(20),
+    IN p_address_id INT,
+    IN p_is_deleted BOOLEAN,
+    IN p_deleted_at DATETIME,
+    IN p_updated_at DATETIME,
```

Comment:
Deze kunnen er ipc uit

Reply:
Eruit gehaald en geupdate in de database